src/flowgger/fuzzer.rs

```diff
@@ -0,0 +1,210 @@
+extern crate quickcheck;
+
+use crate::flowgger;
+
+use quickcheck::QuickCheck;
```

Comment:
all those modules will be imported in prod compilation as well, even thoguh no code. and modules like quickcheck should are only available as dev dependency, so it would fail.